grunttasks/generate-valid-styles.js

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var path = require('path');
+var stripSpecialChars = require('./strip-special-chars');
+var VALID_STYLES_FILE_NAME = 'validStyles.json';
```

Comment:
Include a common config task with the filename declared within. For an example, see how [the content server does it](https://github.com/mozilla/fxa-content-server/blob/master/grunttasks/yeoman.js). Rename the file from camelCase to either snake_case or use-a-dash.